app/stack/route.js

```diff
@@ -14,17 +14,17 @@ export default Ember.Route.extend({
     return tokens.join(' - ');
   },
 
+  setupController: function(controller, model) {
+    controller.set('model', model);
+    controller.set('organizations', this.store.find('organization'));
```

Comment:
That makes sense to me, too. My comment was there to make it clear that the tradeoffs/differences between the approaches so you can choose the one that fits the use case best.